[version-4-6] docs: add 4.6.x patch release notes DOC-2263 (#8225)
spectrocloud/librarium: docs/docs-content/clusters/public-cloud/aws/eks.md

Provisioning an AWS EKS clusters can take several minutes.

> EKS clusters are configured with using the
> [AWS Instance Metadata Service (IMDS)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html)
> configuration specified by the EC2 account defaults. Refer to the
> [Configure the Instance Metadata Service options](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-options.html)
> guide for further information.

Reviewer: ```suggestion
EKS clusters deployed by Palette use the
[AWS Instance Metadata Service (IMDS)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html)
configuration specified in the EC2 account defaults. Refer to the
[Configure the Instance Metadata Service options](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-options.html)
guide for further information.
```
